acceptance/config/beaker/jenkins/el-7-x86_64-mdca.cfg

```diff
@@ -8,6 +8,14 @@ HOSTS:
     platform: el-7-x86_64
     template: Delivery/Quality Assurance/Templates/vCloud/centos-7-x86_64
     hypervisor: vcloud
+    puppetcodedir: /etc/puppetlabs/code
+    distmoduledir: /etc/puppetlabs/code/modules
+    hieraconf:     /etc/puppetlabs/code/hiera.yaml
+    puppetconfdir: /etc/puppetlabs/puppet
+    puppetserver-confdir: /etc/puppetserver/conf.d/
```

Comment:
Is this right?  https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md appears to show `/etc/puppetlabs/puppetserver/conf.d`.